cl/search/templates/includes/order_by_dropdown.html

```diff
@@ -25,6 +25,16 @@
                     selected="selected"
                     {% endif %}>Oldest First</option>
         {% endif %}
+		{% if v == 'r' and request.user.profile.is_tester %}
+			<option value="entry_date_filed desc"
```

Comment:
Can you accept this change in indendtation?

```suggestion
    		{% if v == 'r' and request.user.profile.is_tester %}
		    	<option value="entry_date_filed desc"
```